<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OFInterfaceWidget</class>
 <widget class="QWidget" name="OFInterfaceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>274</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QGroupBox" name="MidiGroupBox">
     <property name="title">
      <string>Ouvert le midi</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_12">
      <item>
       <widget class="QRadioButton" name="OuvertMidiRadio">
        <property name="text">
         <string>Oui</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="FermeMidiRadio">
        <property name="text">
         <string>Non</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="5">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="HorairesMatinGroupBox">
     <property name="title">
      <string>Horaires matin</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="LabelLayout2">
        <item>
         <widget class="QLabel" name="label3">
          <property name="text">
           <string>Ouverture :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label4">
          <property name="text">
           <string>Fermeture :</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="MatinHeureLayout">
        <item>
         <widget class="QSpinBox" name="OMatinHeureSpin">
          <property name="maximum">
           <number>23</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="FMatinHeureSpin">
          <property name="maximum">
           <number>23</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="LabelLayout1">
        <item>
         <widget class="QLabel" name="label1">
          <property name="text">
           <string>h</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label2">
          <property name="text">
           <string>h</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="MatinMinuteLayout">
        <item>
         <widget class="QSpinBox" name="OMatinMinuteSpin">
          <property name="maximum">
           <number>59</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="FMatinMinuteSpin">
          <property name="maximum">
           <number>59</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QGroupBox" name="HoraitesApremGroupBox">
     <property name="title">
      <string>Horaires après-midi</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="LabelLayout3">
        <item>
         <widget class="QLabel" name="label5">
          <property name="text">
           <string>Ouverture :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label6">
          <property name="text">
           <string>Fermeture :</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="ApremHeureLayout">
        <item>
         <widget class="QSpinBox" name="OApremHeureSpin">
          <property name="maximum">
           <number>23</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="FApremHeureSpin">
          <property name="maximum">
           <number>23</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="LabelLayout4">
        <item>
         <widget class="QLabel" name="label8">
          <property name="text">
           <string>h</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Label7">
          <property name="text">
           <string>h</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="ApremMinuteLayout">
        <item>
         <widget class="QSpinBox" name="OApremMinuteSpin">
          <property name="maximum">
           <number>59</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="FApremMinuteSpin">
          <property name="maximum">
           <number>59</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="OFGroupBox">
     <property name="title">
      <string>Magasin</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="OuvertRadio">
        <property name="text">
         <string>Ouvert</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="FermeRadio">
        <property name="text">
         <string>Fermé</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QGroupBox" name="HoraireGroupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Horaires</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="Label6">
        <item>
         <widget class="QLabel" name="label10">
          <property name="text">
           <string>Ouverture :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label9">
          <property name="text">
           <string>Fermeture :</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="HeureLayout">
        <item>
         <widget class="QSpinBox" name="OHeureSpin">
          <property name="maximum">
           <number>23</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="FHeureSpin">
          <property name="maximum">
           <number>23</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="label7">
        <item>
         <widget class="QLabel" name="label12">
          <property name="text">
           <string>h</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label11">
          <property name="text">
           <string>h</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="MinuteLayout">
        <item>
         <widget class="QSpinBox" name="OMinuteSpin">
          <property name="maximum">
           <number>59</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="FMinuteSpin">
          <property name="maximum">
           <number>59</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
